src/CrawlAbstractLvl/private/abslvlworld.cpp

```diff
@@ -31,23 +32,23 @@ CRAWL::WorldRule *AbsLvlWorld::initWorldRules() {
 
     return new CRAWL::WorldRule {
 
-        {0,
+        {200,
             {
-                {registerObject<Baff>(), 10}, {registerObject<CRAWL::DefaultLight>(), 1}
+                {registerObject<ObstacleBlue>(), 10}, {registerObject<CRAWL::DefaultLight>(), 1}
             }
         },
 
-        {20,
+        {300,
             {
-                {registerObject<ObstacleBlue>(), 10}, {registerObject<CRAWL::DefaultLight>(), 1}
+                {registerObject<GroupObstacleRed>(), 1}, {registerObject<CRAWL::DefaultLight>(), 1}
             }
         },
```

Comment:
do you sure that it is right ?
you removed single objects and leave only group objects